.gitignore

```diff
@@ -187,4 +187,9 @@ dmypy.json
 # Protos
 sdk/python/docs/html
 sdk/python/feast/protos/
-sdk/go/protos/
\ No newline at end of file
+sdk/go/protos/
+
+# Local Repos
+innocent_redbird/
```

Comment:
You can use https://www.atlassian.com/git/tutorials/saving-changes/gitignore#personal-git-ignore-rules instead of modifying this gitignore